File: application/app/src/main/java/com/example/hydro/request/models/Game.java
package com.example.hydro.request.models;

public class Game {
    public int id; // : self.id
    public String name; // : self.name
    public String owner; // : self.owner.login
    public String description; // : self.description

    public Game(int id, String name, String owner, String description){
        this.id = id;
        this.name = name;
        this.owner = owner;
        this.description = description;
    }

    public String toString(){
        return  this.name;
    }

    public String getName() {
        return  this.name;
    }
}

File: hydrogen-v2/app/sql/models/game.py
import datetime
import sqlalchemy
import sqlalchemy.orm as orm
import sqlalchemy.ext.declarative as decl

from werkzeug.security import generate_password_hash, check_password_hash
from flask_login import UserMixin

from app.sql.session import SqlAlchemyBase


class Game(SqlAlchemyBase):

	#def defAv():
	#	return open("./static/databases/images/avatars/def.png", "rb").read()

	__tablename__ = "games"

	id = sqlalchemy.Column(
		sqlalchemy.Integer,
		primary_key=True,
		autoincrement=True
	)

	name = sqlalchemy.Column(
		sqlalchemy.String,
		unique = True
	)

	description = sqlalchemy.Column(
		sqlalchemy.String(),
		default="No description"
	)

	owner_id = sqlalchemy.Column(
		sqlalchemy.Integer, sqlalchemy.ForeignKey("users.id")
	)
	owner = orm.relation("User", lazy='subquery')

	def __repr__(self): return f"<Game> {self.id} {self.name}"

	def to_dict(self):
		return {
			"type": "<Game>",
			"id": self.id,
			"name": self.name,
			"description": self.description,
			"owner": self.owner
		}

	def get_json(self):
		return {
			"id": self.id,
			"name": self.name,
			"description": self.description,
			"owner": self.owner.login
		}